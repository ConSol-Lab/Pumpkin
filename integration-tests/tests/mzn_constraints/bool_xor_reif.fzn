var bool: r :: output_var;
var bool: x :: output_var;
var bool: y :: output_var;

constraint pumpkin_bool_xor_reif(x, y, r);

solve satisfy;
