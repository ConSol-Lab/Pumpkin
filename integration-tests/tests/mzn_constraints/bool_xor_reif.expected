r = false;
x = false;
y = false;
----------
r = true;
x = true;
y = false;
----------
r = true;
x = false;
y = true;
----------
r = false;
x = true;
y = true;
----------
==========